Exiled.Events/Patches/Events/Player/Spawning.cs

```diff
@@ -34,11 +34,14 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
 
             // Find the index of the ldarg.0 before the only ldfld CharacterClassManager::SpawnProtected
             const int offset = -1;
-            int index = newInstructions.FindIndex(i => i.opcode == OpCodes.Ldfld && (FieldInfo)i.operand == AccessTools.Field(typeof(CharacterClassManager), nameof(CharacterClassManager.SpawnProtected))) + offset;
+            int index = newInstructions.FindIndex(i =>
+            i.opcode == OpCodes.Call && (MethodInfo)i.operand ==
+            AccessTools.PropertyGetter(typeof(CharacterClassManager), nameof(CharacterClassManager.SpawnProtected))) + offset;
 
             // Remove all existing this._pms.OnPlayerClassChange calls (we will want to call this ourselves after our even fires, to allow their spawn position to change.)
-            newInstructions.RemoveAll(i =>
-                i.opcode == OpCodes.Call && (MethodInfo)i.operand == AccessTools.Method(typeof(PlayerMovementSync), nameof(PlayerMovementSync.OnPlayerClassChange)));
+            foreach (CodeInstruction instruction in newInstructions.FindAll(i =>
```

Comment:
Uh, 5.3 branch seems to have gone backwards on this, bad?

Reply:
There are a lot of changes that do not apply to your PR, and are in other branches in Exiled, yea, bad, revert all these changes